sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -204,14 +162,27 @@ export interface ReceivedMessageInfo extends SendableMessageInfo {
     readonly sequenceNumber?: Long;
 }
 
+// @public
+export interface ReceivedSettleableMessage extends ReceivedMessage {
```

Comment:
I really really really don't like this name
- hard to read
- "settle" as a term is not used anywhere else in the api surface
- "settle" can be a new term for a segment of users that they now need to be aware of

How about `ReceivedLockedMessage`?

The term "locked" can be more familiar to the user. They received a message that has been locked for them. The lock is released by the complete/abandon/deadLetter/defer methods, the lock can be renewed by the `renewLock` method.

Reply:
Sure :)